doc/callbacks.md

```diff
@@ -8,53 +8,74 @@ as:
 
 Callbacks can be passed as an argument to the `Optimize()` function:
 
+<details>
+<summary>Click to check out the example</summary>
+<p>
+
 ```c++
-RosenbrockFunction f;
-arma::mat coordinates = f.GetInitialPoint();
+  RosenbrockFunction f;
+  arma::mat coordinates = f.GetInitialPoint();
```

Comment:
I don't think the spacing changes are needed. :+1:

Reply:
:) adapted